boards/arm/stm32f429i_disc1/doc/stm32f429i_disc1.rst

```diff
@@ -0,0 +1,203 @@
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on Zephyr porting.
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/stm32f429i_disc1/stm32f429i_disc1_defconfig``
+
+
+Pin Mapping
+===========
+
+STM32F429I-DISC1 Discovry kit has 8 GPIO controllers. These controllers are responsible for pin muxing,
+input/output, pull-up, etc.
+
+For mode details please refer to `STM32F429-DISC1 board User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+- UART_1_TX : PA9
+- UART_1_RX : PA10
+- UART_2_TX : PA2
+- UART_2_RX : PA3
+- USER_PB : PA0
+- LD3 : PG13
+- LD4 : PG12
+
+System Clock
+============
+
+STM32F429I-DISC1 System Clock could be driven by internal or external oscillator,
+as well as main PLL clock. By default System clock is driven by PLL clock at 168MHz,
+driven by 8MHz high speed external clock.
+
+Serial Port
+===========
+
+STM32F429I-DISC1 Discovery kit has up to 8 UARTs. The Zephyr console output is assigned to UART1.
+Default settings are 115200 8N1.
+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+STM32F429I-DISC1 Discovery kit includes an ST-LINK/V2-B embedded debug tool interface.
+This interface is supported by the openocd version included in Zephyr SDK.
+
+Flashing an application to STM32F4DISCOVERY
+-------------------------------------------
+
+The sample application :ref:`hello_world` is being used in this tutorial:
+
+.. code-block:: console
+
+   $<zephyr_root_path>/samples/hello_world
+
+To build the Zephyr kernel and application, enter:
+
+.. code-block:: console
+
+   $ cd <zephyr_root_path>
+   $ source zephyr-env.sh
+   $ cd $ZEPHYR_BASE/samples/hello_world/
+   $ make BOARD=stm32f429i_disc1
+
+Connect the STM32F429I-DISC1 Discovery kit to your host computer using the USB port.
+Then, enter the following command:
+
+.. code-block:: console
+
+   $ make BOARD=stm32f429i_disc1 flash
+
+Run a serial host program to connect with your board:
+
+.. code-block:: console
+
+   $ minicom -D /dev/ttyACM0
+
+You should see the following message:
+
+.. code-block:: console
+
+   Hello World! arm
+
+
+Debugging
+=========
+
+Access gdb with the following make command:
+
+.. code-block:: console
+
+   $ make BOARD=stm32f429i_disc1 debug
+
+
+.. _STM32F429I-DISC1 website:
+   http://www.st.com/en/evaluation-tools/32f429idiscovery.html
+
+.. _STM32F429I-DISC1 board User Manual:
+   http://www.st.com/web/en/resource/technical/document/user_manual/DM00097320.pdf
+
+.. _STM32F429ZI on www.st.com:
+   http://www.st.com/en/microcontrollers/stm32f429-439.html
+
+..
```

Comment:
looks like a stray ".." at the end of the file (you're missing three link targets referenced earlier in the doc)